feat: add `blas/base/drotmg`

ShabiShett07 opened the pull request:
Progresses: #2039.

## Description

> What is the purpose of this pull request?

This pull request:

-  Progresses: #2039

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   progresses #2039 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

ShabiShett07 committed:
feat: add /blas/base/drotmg

ShabiShett07 committed:
fix: multiple errors

ShabiShett07 commented:
/stdlib update-copyright-years

stdlib-bot committed:
chore: update copyright years

ShabiShett07 commented:
This PR is ready for review, as I have fixed the errors that it had previously

ShabiShett07 committed:
fix: update test files

ShabiShett07 committed:
fix: update assign.js

Signed-off-by: Shabareesh Shetty <139731143+ShabiShett07@users.noreply.github.com>

ShabiShett07 committed:
fix: update index.js

Signed-off-by: Shabareesh Shetty <139731143+ShabiShett07@users.noreply.github.com>

ShabiShett07 committed:
fix: update test.ts

Signed-off-by: Shabareesh Shetty <139731143+ShabiShett07@users.noreply.github.com>

ShabiShett07 committed:
fix:  update README

ShabiShett07 committed:
fix:  update examples

ShabiShett07 committed:
fix: update test.assign.js

Signed-off-by: Shabareesh Shetty <139731143+ShabiShett07@users.noreply.github.com>

ShabiShett07 committed:
fix: update test.main.js

ShabiShett07 committed:
fix: update README.md

Signed-off-by: Shabareesh Shetty <139731143+ShabiShett07@users.noreply.github.com>

Planeshifter commented:
<!-- stdlib-bot-tracking-issue-closure-check -->
:warning: **Tracking Issue Closure Warning** :warning:

I noticed your PR description contains closing keywords ("Resolves", "Closes", or "Fixes") referencing a "Tracking Issue".

**Why this matters:**
Tracking issues should typically remain open until all related sub-issues are completed. GitHub automatically closes issues with such closing keywords when the PR is merged.

**Required action:**
Use "Progresses:" or "Ref:" instead to reference the tracking issue without automatically closing it.

Thank you for your contribution to the project!

ShabiShett07 commented:
/stdlib merge

stdlib-bot committed:
Merge remote-tracking branch 'upstream/develop' into feature/drotmg

ShabiShett07 committed:
fix: indentation

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: passed
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: na
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: passed
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

ShabiShett07 commented:
/stdlib merge

stdlib-bot committed:
Merge remote-tracking branch 'upstream/develop' into feature/drotmg

ShabiShett07 commented:
/stdlib merge

stdlib-bot committed:
Merge remote-tracking branch 'upstream/develop' into feature/drotmg
